pykeybasebot/errors.py

```diff
@@ -7,8 +10,22 @@ def __str__(self):
 
 
 class RevisionError(Error):
-    pass
+    CODE = 2760
 
 
 class DeleteNonExistentError(Error):
-    pass
+    CODE = 2762
+
+
+def try_to_error(e: Exception) -> Union[Exception, Error]:
```

Comment:
this function is in a pretty high level namespace (i.e. it's in a file called `errors` in the root of `pykeybasebot`). how do you feel about calling it something more specific, so it's not confusing for the next client that wants to implement something similar? 
maybe something like: `disambiguate_kvstore_error`?

Reply:
the json error codes should uniquely correspond to all (and any future) custom Error classes right? i don't see strong reason to name this method to be kvstore specific, but lmk what you think of the name `disambiguate_error`